generators/fix-entity/index.js

```diff
@@ -116,12 +113,15 @@ module.exports = generator.extend({
          */
         const getLiquibaseFile = type => `${jhipsterVar.resourceDir}config/liquibase/changelog/${this.entityConfig.data.changelogDate}_added_${type}_${this.entityConfig.entityClass}.xml`;
 
+        // TODO: freeze object (safely)
         const files = {
             config: this.entityConfig.filename,
             ORM: `${jhipsterVar.javaDir}domain/${this.entityConfig.entityClass}.java`,
             liquibaseEntity: getLiquibaseFile('entity')
         };
 
+        const filesArr = Object.keys(files);
```

Comment:
It takes as parameter this very previous assignment to `files`, I don't think so.